kratos/CMakeLists.txt

```diff
@@ -86,6 +86,12 @@ if(${USE_TRIANGLE_NONFREE_TPL} MATCHES ON )
   set(KRATOS_EXTERNAL_LIBRARIES ${KRATOS_EXTERNAL_LIBRARIES} triangle)
 endif(${USE_TRIANGLE_NONFREE_TPL} MATCHES ON )
 
+if(${CMAKE_COMPILER_IS_GNUCXX})
```

Comment:
Try to avoid this using my suggestion

Reply:
This cannot be avoided, see the end of the documentation https://en.cppreference.com/w/cpp/filesystem
GCC initially had filesystem implemented in a separate lib (until v9), hence before that it has to be linked explicitly
Hence I am quite sure your suggestion wouldnt change this

Overall I think filesystem is quite well integrated by now, see the compiler support: https://en.cppreference.com/w/cpp/compiler_support/17
Hence unless someone has problems I would leave the include as default for now

Reply:
Okay, this is my personal experience

Reply:
which systems/compilers did you have problems with?

Reply:
Clang in Mac and different versions of GCC

Reply:
ok good to know

Please create issues in case it causes problems